lib/bolt/outputter/human.rb

```diff
@@ -93,6 +93,21 @@ def print_table(results)
         )
       end
 
+      # @param [Hash] A hash representing the task
+      def print_task(task)
+        pretty_params = ""
+        task_info = ""
+        task['parameters'].each do |k, v|
+          pretty_params << "- #{k}: #{v['type']}\n\t#{v['description']}\n"
```

Comment:
This feels gross but also works and may be the best way to build the string?

Reply:
It seems fine. You could use heredoc, but I think that'd be less clear if we actually want tabs.

Reply:
It will print an extra \n\t when there is no description which is probably wrong.